### DISCUSSION TITLE
"Save_Terminal" oddities

### DISCUSSION THREAD

**Discussion by ghramsey** [MattFiler/LegendPlugin#93] [Q&A] [answered]
I tried moving the save terminal in the Tech Security station to the room w/ the corpse that Amanda reacts to.
I could hear it, but it was not visible.  I used Cheat Engine clipping and it was not visible inside any of the walls nor ceiling or floor.
So I moved it to the center of the room.  I could see if from the area where the tables in the outer room are.  
As I approached the room though the save station vanished.  The beeps however were still there, but nothing else was.    
I cloned the save station and renamed it to be a new save point and restored the original location parameters.   
The new spawned save point works fine.

I've had similar issue before with entities named "Save_Terminal".  
I created a new save station in mission 5 and tried to remove the one by the door to Kuhlman's lobby room. 
The new station was fine.  The removed station was gone from the environment composite but the physical 
entity was still present but w/o it's UI.

Any ideas what might be going on?  It only happens if I move or remove the original save stations named
 "Save_Terminal_ "  leading me to think something about those entity names is required.  (?) or something else is afoot.
 
 Any insight into this is helpful.

**Comment by MattFiler** [comment, marked as answer]
This will be related to zoning - I imagine that the save terminal is in the zone for the security section, which is then unloaded when you go to the other area, thus making it invisible.

Make sure you move the terminal into the correct TriggerSequence executed by the Zone node for the room you've moved it to, and then it will load in at the appropriate times.
